gateway/requirements.txt

```diff
@@ -1,3 +1,4 @@
+cryptography>=42.0.5
```

Comment:
For what do we need this dependency, Aki?

Reply:
I have no idea why this started happening.

Reply:
Oh, I saw the error in PR that I just opened. I will take a look 👀

Reply:
yep, I also noticed